cmd/nerdctl/run_cgroup_linux_test.go

```diff
@@ -283,6 +283,50 @@ func TestRunCgroupConf(t *testing.T) {
 		"cat", "memory.high").AssertOutExactly("33554432\n")
 }
 
+func TestRunCgroupParent(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	info := base.Info()
+	containerName := testutil.Identifier(t)
+	defer base.Cmd("rm", "-f", containerName).Run()
+
+	switch info.CgroupDriver {
+	case "none", "":
+		t.Skip("test requires cgroup driver")
+	}
+
+	parent := "/foobarbaz"
+	if info.CgroupDriver == "systemd" {
```

Comment:
Can you log out somewhere which cgroup drivers are being run on the github host to understand which path are we testing?

Reply:
Sure, and I'll look/address the other feedback sometime this weekend. Thanks!

Reply:
Added in this test